### DISCUSSION TITLE
Getting stated...

### DISCUSSION THREAD

**Discussion by skipster74** [xblax/flashforge_ad5m_klipper_mod#177] [Q&A]
A special thanks to everyone for making this happen. 

I just tried my first install. Went well enough but when I try to actually print, it err'd out saying that the nozzel tempeture was too low. When I tried to fix it in the base config I now get an ADC "out of range"

The one thing I noticed was that it was set to -100 in the file. No matter what i change it to, it still crashes. 0, 160, 170. Doesn't matter. 

I'm sure i'm doing something very dumb/stupid, so thanks for putting up with this in advanced. I feel So close, yet SO far away.

b

**Comment by KaruroChori** [comment]
Hi, have you followed these instructions?
https://github.com/xblax/flashforge_ad5m_klipper_mod/blob/master/docs/SLICING.md#printer-profile

**Comment by MrSolidus89** [comment]
However, I don't understand why it repeats the purge line 2 times, I use Orca, I have already changed the start and end, do you have a solution?

**Reply by KaruroChori** [reply]
Can you share a small gcode file sliced with that profile & which resuls in the two purge line issue?

**Reply by MrSolidus89** [reply]
here is a small test file
[Touch_pen_PLA_56m43s.zip](https://github.com/xblax/flashforge_ad5m_klipper_mod/files/15251871/Touch_pen_PLA_56m43s.zip)

**Reply by KaruroChori** [reply]
Your start gcode has a wiping process right after you call START_PRINT. You have not changed the configuration in the slicer properly.

**Reply by MrSolidus89** [reply]
![Screenshot 2024-05-07 215711](https://github.com/xblax/flashforge_ad5m_klipper_mod/assets/55961986/2fb87110-7ca4-4540-873f-e474b6c9a5e1)

**Reply by MrSolidus89** [reply]
I only added the two strings that the document said, everything else was left untouched

**Reply by KaruroChori** [reply]
Exactly. Everything else should not be there. It it already accounted for in the custom macros. This way you are just duplicating it.

**Reply by MrSolidus89** [reply]
perfect I understand, I deleted everything leaving only those two commands, in practice it recalls the macros, now I understand thank you very much

**Comment by skipster74** [comment]
Thanks again for this great community.

Still can't print. Uploads fine and the bed heats up to the correct temp but the extruder stays dormant. 

After a few moments after reaching the bed tempeture, the hotend will register that it's being asked to go to 245 but then just sits there and does nothing. I assume it's something in the macro/config but wanted to ask before I did more damage.

**Reply by consp** [reply]
> I assume it's something in the macro/config

Very much looks like it. Do the interfaces (fluidd/mainsail) show it's trying to heat up? I'd guess it's doing a heat-but-not-wait (M104 instead of M109) and then stops due to it not being warm enough yet. Could you post the first lines of the actual uploaded gcode file here (or the entire file)?
